modules/appnexusBidAdapter.js

```diff
@@ -36,7 +36,7 @@ const SOURCE = 'pbjs';
 const MAX_IMPS_PER_REQUEST = 15;
 const mappingFileUrl = '//acdn.adnxs.com/prebid/appnexus-mapping/mappings.json';
 const SCRIPT_TAG_START = '<script';
-const VIEWABILITY_URL_START = /http:\/\/cdn\.adnxs\.com\/v/;
+const VIEWABILITY_URL_START = /https:\/\/cdn\.adnxs\.com\/v/;
```

Comment:
This does not need to be changed, we actually need the regex to not include the protocol.  We need to change the protocol for this script later in the adapter code so that when it's written to the page, it'll use `https`.

When the bid returns from the adserver, the default viewability script will be using a protocol that's read from the page.  So it would come in as either `http` or `https`.  Because of this variability, the regex needs to be non-specific.